src/psygnal/_evented_model_utils.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from types import TracebackType
+
+    import pydantic.version
+
+    if pydantic.version.VERSION.startswith("2"):
+        from ._evented_model_v2 import EventedModel as EventedModelv1
```

Comment:
Nit: I would have expected `EventedModelV1` and `EventedModelV2` (i.e. with an upper case V, not lower case v).